docs/nia: Update status API docs to table format (#11927)
hashicorp/consul: website/content/docs/nia/api/status.mdx

#### Request Parameters
Currently no request parameters are offered for the overall status API.

#### Response Statuses

| Status | Reason           |
| ------ | ---------------- |
| 200    | Successfully retrieved the status |

#### Response Fields

| Name	| Type	 | Description        |
| ----- | ------ | ------------------ |

> |`status` | object |  Summary count of how many tasks have each health status value.

Reviewer: ```suggestion
|`status` | object |  The summary count of how many tasks that each have a health status value.
```
Cleaner?

Author: I think this implies that not all tasks have a health status, which isn't the case. Maybe something like `The summary of how many tasks have a given health status value`?

#### Request Parameters

| Name	   | Type	  | Description        |
| -------- | ------ | ------------------ |
|`task`    | string | Option to specify the name of the task to return in the response. If not specified, all tasks are returned in the response.
|`include` | string | Only accepts the value `"events"`. Use to include stored event information in response.
|`status`  | string | Only accepts health status values `"successful"`, `"errored"`, `"critical"`, or `"unknown"`. Use to filter response by tasks that have the specified health status value. Recommend setting this parameter when requesting all tasks i.e. no `task` parameter is set.

#### Response Statuses

| Status | Reason           |
| ------ | ---------------- |
| 200  | Successfully retrieved the task status |
| 404  | Task with the given name not found |

#### Response Fields

The response is a JSON map of task name to a status information structure with the following fields.

| Name	      | Type	  | Description       |
| ----------- | ------ | ------------------ |

> |`task_name`  | string | Name that task is configured with in Consul-Terraform-Sync.

Reviewer: ```suggestion
|`task_name`  | string | Name of the task as configured in Consul-Terraform-Sync.
```
Cleaner?
